public/security_analytics_app.tsx

```diff
@@ -45,44 +30,13 @@ export function renderApp(
   depsStart: SecurityAnalyticsPluginStartDeps,
   dataSourceManagement?: DataSourceManagementPluginSetup
 ) {
-  const { http, savedObjects } = coreStart;
-
-  const detectorsService = new DetectorsService(http);
-  const correlationsService = new CorrelationService(http);
-  const indexService = new IndexService(http);
-  const findingsService = new FindingsService(http, coreStart.notifications);
-  const opensearchService = new OpenSearchService(http, savedObjects.client);
-  const fieldMappingService = new FieldMappingService(http);
-  const alertsService = new AlertsService(http, coreStart.notifications);
-  const ruleService = new RuleService(http);
-  const notificationsService = new NotificationsService(http);
-  const savedObjectsService = new SavedObjectService(savedObjects.client, indexService);
-  const indexPatternsService = new IndexPatternsService(depsStart.data.indexPatterns);
-  const logTypeService = new LogTypeService(http);
-  const metricsService = new MetricsService(http);
-  const threatIntelService = new ThreatIntelService(http, coreStart.notifications);
-
-  const services: BrowserServices = {
-    detectorsService,
-    correlationsService,
-    indexService,
-    fieldMappingService,
-    findingsService,
-    opensearchService,
-    ruleService,
-    alertService: alertsService,
-    notificationsService,
-    savedObjectsService,
-    indexPatternsService,
-    logTypeService,
-    metricsService,
-    threatIntelService,
-  };
+  const metricsService = new MetricsService(coreStart.http);
```

Comment:
We should remove this and get it using services itself like we get notificationsService below

Reply:
Changed to use `services.metricsService`